libuuu/cmd.cpp

```diff
@@ -289,6 +298,7 @@ CmdObjCreateMap::CmdObjCreateMap()
 
 	(*this)["_ALL:DONE"] = new_cmd_obj<CmdDone>;
 	(*this)["_ALL:DELAY"] = new_cmd_obj<CmdDelay>;
+        (*this)["_ALL:CONFIG"] = new_cmd_obj<CmdConfig>;
```

Comment:
Are you sure all protocol need config command.
 
There are already have CFG: protocol to set PID VID to protocol map? 

which will run once at beginning. 
 
CFG: HID -maxtimeout 1000

Reply:
what is CFG protocol? this isn't documented... I don't understand your remark....

Reply:
in help
     
     CFG:  Config protocol of specific usb device vid/pid
                SDPS|SDP|FB\Fastboot|FBK -chip <chip name> -pid <pid> -vid <vid